javascript/ql/src/semmle/javascript/security/dataflow/TaintedPathCustomizations.qll

```diff
@@ -682,6 +682,21 @@ module TaintedPath {
     }
   }
 
+  /**
+   * The `cwd` option for the `read-pkg` library.
+   */
+  private class ReadPkgCwdSink extends TaintedPath::Sink {
+    ReadPkgCwdSink() {
+      this =
+        API::moduleImport("read-pkg")
+            .getMember(["readPackageAsync", "readPackageSync"])
+            .getACall()
```

Comment:
```suggestion
```
`getACall` should not be a needed here (unless you specifically want to avoid `new`-calls).